src/main/java/edu/kit/provideq/toolbox/meta/SolvingProperties.java

```diff
@@ -0,0 +1,30 @@
+package edu.kit.provideq.toolbox.meta;
+
+import edu.kit.provideq.toolbox.meta.setting.SolverSetting;
+import java.util.List;
+import java.util.Optional;
+
+public final class SolvingProperties {
+  private final List<SolverSetting> settings;
+
+  public SolvingProperties(
+      List<SolverSetting> settings
+  ) {
+    this.settings = settings;
+  }
+
+  @SuppressWarnings("unchecked") // caller must ensure correct type
```

Comment:
The note that the caller must ensure the correct type should be noted in a Java Doc of this method.
This small comment might get unnoted when the source code of this method is not checked directly.